docs/madr/decisions/009-tracing-policy.md

```diff
@@ -0,0 +1,184 @@
+# Traffic tracing policy
+
+- Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/4732
+
+## Context and Problem Statement
+
+We want to create a [new policy matching compliant](https://github.com/kumahq/kuma/blob/22c157d4adac7f518b1b49939c7e9ea4d2a1876c/docs/madr/decisions/005-policy-matching.md)
+resource for managing traffic tracing.
+
+## Decision Drivers
+
+- Replace existing policies with new policy matching compliant resources
+- [Path/Method filtering on traces](https://github.com/kumahq/kuma/issues/3335)
+- [Custom tags](https://github.com/kumahq/kuma/issues/3275)
+- Not blocking [OpenTelemetry tracing](https://github.com/kumahq/kuma/issues/3690)
+
+## Considered Options
+
+The new resource will be called `Tracing` in the rest of this document.
+
+Tracing in Kuma is implemented with Envoy's [tracing
+support](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing).
+At the moment, Kuma supports Zipkin and Datadog.
+
+### Naming
+
+Some potential names:
+
+- `Tracing`
+- `Traces`
+- `MeshTrafficTrace`
```

Comment:
+1 to that

Reply:
+1

Reply:
+1

side note: erh... The plural is killing everything 😠 `MeshTracings` 😢